packages/contentful--react-apps-toolkit/src/useAutoResizer.tsx

```diff
@@ -0,0 +1,17 @@
+import { FieldExtensionSDK } from '@contentful/app-sdk';
+import { useEffect } from 'react';
+import { useSDK } from './useSDK';
+
+/**
+ * Invokes the FieldExtensionSDK autoResizer.
+ * Must be wrapped by SDKProvider.
+ * Can only be used when app is rendered in field, sidebar or entry editor location.
+ */
+export function useAutoResizer(): void {
+  const sdk = useSDK<FieldExtensionSDK>();
+
```

Comment:
We should check whether `sdk.window` is available and throw a pretty error message here. Otherwise people will receive a generic "trying to access `startAutoResizer` on `undefined`" message.

Reply:
Hey @andipaetzold, I have implemented this. Let me know your thoughts on the message.